ui/css/perf.css

```diff
@@ -206,6 +206,14 @@ h1 {
 
 /* Compare interface stuff */
 
+.permalink {
+  margin-left: 5px;
```

Comment:
You might use the bootstrap spacing classes directly on the element (something like "ml-1" or "ml-2" for a little margin-left)
https://getbootstrap.com/docs/4.3/utilities/spacing/